2020-12-20T17:01:35.3910241+00:00  [INF] Entity Framework Core "3.1.8" initialized '"CoreContext"' using provider '"Pomelo.EntityFrameworkCore.MySql"' with options: "None" (9958f5bb)
2020-12-20T17:01:35.9399526+00:00  [INF] Executed DbCommand ("21"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT CASE WHEN COUNT(*) = 0 THEN FALSE ELSE TRUE END
FROM information_schema.tables
WHERE table_type = 'BASE TABLE' AND table_schema = 'mutant_test_db'" (0723d8ff)
2020-12-20T17:01:36.1282132+00:00  [INF] Now listening on: "http://[::]:80" (d826f4b8)
2020-12-20T17:01:36.1322921+00:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-20T17:01:36.1358895+00:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-20T17:01:36.1397728+00:00  [INF] Content root path: "/app" (b5d60022)
2020-12-20T17:01:36.9429475+00:00 0HM5513KNVGOQ:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:32772/swagger/index.html   (ca22a1cb)
2020-12-20T17:01:37.0576973+00:00 0HM5513KNVGOQ:00000001 [INF] Request finished in 117.6142ms 200 text/html;charset=utf-8 (791a596a)
2020-12-20T17:01:37.3807401+00:00 0HM5513KNVGOQ:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:32772/swagger/v1/swagger.json   (ca22a1cb)
2020-12-20T17:01:37.5304259+00:00 0HM5513KNVGOQ:00000002 [INF] Request finished in 149.9324ms 200 application/json;charset=utf-8 (791a596a)
2020-12-20T17:01:43.1187072+00:00 0HM5513KNVGOQ:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:32772/users/download   (ca22a1cb)
2020-12-20T17:01:43.1706379+00:00 0HM5513KNVGOQ:00000003 [INF] Executing endpoint '"MutantTest.API.Controllers.UsersController.GetUsers (MutantTest.API)"' (500cc934)
2020-12-20T17:01:43.2166102+00:00 0HM5513KNVGOQ:00000003 [INF] Route matched with "{action = \"GetUsers\", controller = \"Users\"}". Executing controller action with signature "System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[MutantTest.API.Controllers.Dto.UserInfoDTO]]] GetUsers()" on controller "MutantTest.API.Controllers.UsersController" ("MutantTest.API"). (122b2fdf)
2020-12-20T17:01:43.2239672+00:00 0HM5513KNVGOQ:00000003 [INF] Entity Framework Core "3.1.8" initialized '"CoreContext"' using provider '"Pomelo.EntityFrameworkCore.MySql"' with options: "None" (9958f5bb)
2020-12-20T17:01:43.4077456+00:00 0HM5513KNVGOQ:00000003 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T17:01:43.4153727+00:00 0HM5513KNVGOQ:00000003 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T17:01:43.5990014+00:00 0HM5513KNVGOQ:00000003 [INF] Received HTTP response after 171.7511ms - OK (6350df9e)
2020-12-20T17:01:43.6048150+00:00 0HM5513KNVGOQ:00000003 [INF] End processing HTTP request after 219.5248ms - OK (7656b38e)
2020-12-20T17:01:43.8144991+00:00 0HM5513KNVGOQ:00000003 [INF] Lista usuarios recuperada com sucesso. (0a6899ef)
2020-12-20T17:01:43.8235240+00:00 0HM5513KNVGOQ:00000003 [INF] Executing ObjectResult, writing value of type '"System.Collections.Generic.List`1[[MutantTest.API.Controllers.Dto.UserInfoDTO, MutantTest.API, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]"'. (8a1b66c8)
2020-12-20T17:01:43.8467702+00:00 0HM5513KNVGOQ:00000003 [INF] Executed action "MutantTest.API.Controllers.UsersController.GetUsers (MutantTest.API)" in 622.1784ms (afa2e885)
2020-12-20T17:01:43.8476724+00:00 0HM5513KNVGOQ:00000003 [INF] Executed endpoint '"MutantTest.API.Controllers.UsersController.GetUsers (MutantTest.API)"' (99874f2b)
2020-12-20T17:01:43.8514201+00:00 0HM5513KNVGOQ:00000003 [INF] Request finished in 733.7744ms 200 application/json; charset=utf-8 (791a596a)
2020-12-20T17:02:26.4454552+00:00 0HM5513KNVGOQ:00000004 [INF] Request starting HTTP/1.1 POST http://localhost:32772/users/save  0 (ca22a1cb)
2020-12-20T17:02:26.4488795+00:00 0HM5513KNVGOQ:00000004 [INF] Executing endpoint '"MutantTest.API.Controllers.UsersController.SaveUsers (MutantTest.API)"' (500cc934)
2020-12-20T17:02:26.4676953+00:00 0HM5513KNVGOQ:00000004 [INF] Route matched with "{action = \"SaveUsers\", controller = \"Users\"}". Executing controller action with signature "System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[MutantTest.API.Controllers.Dto.UserInfoDTO]]] SaveUsers()" on controller "MutantTest.API.Controllers.UsersController" ("MutantTest.API"). (122b2fdf)
2020-12-20T17:02:26.5161407+00:00 0HM5513KNVGOQ:00000004 [INF] Entity Framework Core "3.1.8" initialized '"CoreContext"' using provider '"Pomelo.EntityFrameworkCore.MySql"' with options: "None" (9958f5bb)
2020-12-20T17:02:26.5303359+00:00 0HM5513KNVGOQ:00000004 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T17:02:26.5344466+00:00 0HM5513KNVGOQ:00000004 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T17:02:26.5889685+00:00 0HM5513KNVGOQ:00000004 [INF] Received HTTP response after 52.9109ms - OK (6350df9e)
2020-12-20T17:02:26.5936976+00:00 0HM5513KNVGOQ:00000004 [INF] End processing HTTP request after 63.2599ms - OK (7656b38e)
2020-12-20T17:02:27.1116650+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("24"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.1386085+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("1"ms) [Parameters=["@p0='?' (DbType = Decimal), @p1='?' (DbType = Decimal)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p0, @p1);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.1502612+00:00 0HM5513KNVGOQ:00000004 [ERR] Failed executing DbCommand ("8"ms) [Parameters=["@p0='?' (Size = 255), @p1='?' (Size = 4000), @p2='?' (DbType = Int32), @p3='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p0, @p1, @p2, @p3);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (627a98df)
2020-12-20T17:02:27.2786050+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.CreateAsync(CommandListPosition commandListPosition, ICommandPayloadCreator payloadCreator, IDictionary`2 cachedProcedures, IMySqlCommand command, CommandBehavior behavior, IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 391
   at MySqlConnector.Core.CommandExecutor.ExecuteReaderAsync(IReadOnlyList`1 commands, ICommandPayloadCreator payloadCreator, CommandBehavior behavior, IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/Core/CommandExecutor.cs:line 62
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlCommand.cs:line 210
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.CreateAsync(CommandListPosition commandListPosition, ICommandPayloadCreator payloadCreator, IDictionary`2 cachedProcedures, IMySqlCommand command, CommandBehavior behavior, IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 391
   at MySqlConnector.Core.CommandExecutor.ExecuteReaderAsync(IReadOnlyList`1 commands, ICommandPayloadCreator payloadCreator, CommandBehavior behavior, IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/Core/CommandExecutor.cs:line 62
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlCommand.cs:line 210
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:27.4360080+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Shanna@melissa.tv (62268090)
2020-12-20T17:02:27.4551243+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (DbType = Decimal), @p7='?' (DbType = Decimal), @p8='?' (DbType = Decimal), @p9='?' (DbType = Decimal), @p10='?' (Size = 255), @p11='?' (Size = 4000), @p12='?' (DbType = Int32), @p13='?' (Size = 4000), @p14='?' (Size = 255), @p15='?' (Size = 4000), @p16='?' (DbType = Int32), @p17='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p3, @p4, @p5);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p6, @p7);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p8, @p9);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p10, @p11, @p12, @p13);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p14, @p15, @p16, @p17);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.4669175+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:27.5478011+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Nathan@yesenia.net (45d3b3f6)
2020-12-20T17:02:27.5567385+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (Size = 4000), @p9='?' (DbType = Decimal), @p10='?' (DbType = Decimal), @p11='?' (DbType = Decimal), @p12='?' (DbType = Decimal), @p13='?' (DbType = Decimal), @p14='?' (DbType = Decimal), @p15='?' (Size = 255), @p16='?' (Size = 4000), @p17='?' (DbType = Int32), @p18='?' (Size = 4000), @p19='?' (Size = 255), @p20='?' (Size = 4000), @p21='?' (DbType = Int32), @p22='?' (Size = 4000), @p23='?' (Size = 255), @p24='?' (Size = 4000), @p25='?' (DbType = Int32), @p26='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p3, @p4, @p5);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p6, @p7, @p8);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p9, @p10);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p11, @p12);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p13, @p14);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p15, @p16, @p17, @p18);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p19, @p20, @p21, @p22);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p23, @p24, @p25, @p26);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.5628215+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:27.6479082+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Lucio_Hettinger@annie.ca (cd56b31c)
2020-12-20T17:02:27.6588088+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (Size = 4000), @p9='?' (Size = 4000), @p10='?' (Size = 4000), @p11='?' (Size = 4000), @p12='?' (DbType = Decimal), @p13='?' (DbType = Decimal), @p14='?' (DbType = Decimal), @p15='?' (DbType = Decimal), @p16='?' (DbType = Decimal), @p17='?' (DbType = Decimal), @p18='?' (DbType = Decimal), @p19='?' (DbType = Decimal), @p20='?' (Size = 255), @p21='?' (Size = 4000), @p22='?' (DbType = Int32), @p23='?' (Size = 4000), @p24='?' (Size = 255), @p25='?' (Size = 4000), @p26='?' (DbType = Int32), @p27='?' (Size = 4000), @p28='?' (Size = 255), @p29='?' (Size = 4000), @p30='?' (DbType = Int32), @p31='?' (Size = 4000), @p32='?' (Size = 255), @p33='?' (Size = 4000), @p34='?' (DbType = Int32), @p35='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p3, @p4, @p5);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p6, @p7, @p8);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p9, @p10, @p11);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p12, @p13);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p14, @p15);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p16, @p17);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p18, @p19);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p20, @p21, @p22, @p23);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p24, @p25, @p26, @p27);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p28, @p29, @p30, @p31);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p32, @p33, @p34, @p35);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.6872211+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:27.7705208+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Telly.Hoeger@billy.biz (f62dd99c)
2020-12-20T17:02:27.8110904+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (Size = 4000), @p9='?' (Size = 4000), @p10='?' (Size = 4000), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 4000), @p14='?' (Size = 4000), @p15='?' (DbType = Decimal), @p16='?' (DbType = Decimal), @p17='?' (DbType = Decimal), @p18='?' (DbType = Decimal), @p19='?' (DbType = Decimal), @p20='?' (DbType = Decimal), @p21='?' (DbType = Decimal), @p22='?' (DbType = Decimal), @p23='?' (DbType = Decimal), @p24='?' (DbType = Decimal), @p25='?' (Size = 255), @p26='?' (Size = 4000), @p27='?' (DbType = Int32), @p28='?' (Size = 4000), @p29='?' (Size = 255), @p30='?' (Size = 4000), @p31='?' (DbType = Int32), @p32='?' (Size = 4000), @p33='?' (Size = 255), @p34='?' (Size = 4000), @p35='?' (DbType = Int32), @p36='?' (Size = 4000), @p37='?' (Size = 255), @p38='?' (Size = 4000), @p39='?' (DbType = Int32), @p40='?' (Size = 4000), @p41='?' (Size = 255), @p42='?' (Size = 4000), @p43='?' (DbType = Int32), @p44='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p3, @p4, @p5);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p6, @p7, @p8);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p9, @p10, @p11);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p12, @p13, @p14);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p15, @p16);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p17, @p18);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p19, @p20);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p21, @p22);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p23, @p24);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p25, @p26, @p27, @p28);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p29, @p30, @p31, @p32);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p33, @p34, @p35, @p36);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p37, @p38, @p39, @p40);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p41, @p42, @p43, @p44);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.8283709+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Shanna@melissa.tv' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:27.8873526+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Sherwood@rosamond.me (30f174b5)
2020-12-20T17:02:27.9061774+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (Size = 4000), @p9='?' (Size = 4000), @p10='?' (Size = 4000), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 4000), @p14='?' (Size = 4000), @p15='?' (Size = 4000), @p16='?' (Size = 4000), @p17='?' (Size = 4000), @p18='?' (DbType = Decimal), @p19='?' (DbType = Decimal), @p20='?' (DbType = Decimal), @p21='?' (DbType = Decimal), @p22='?' (DbType = Decimal), @p23='?' (DbType = Decimal), @p24='?' (DbType = Decimal), @p25='?' (DbType = Decimal), @p26='?' (DbType = Decimal), @p27='?' (DbType = Decimal), @p28='?' (DbType = Decimal), @p29='?' (DbType = Decimal), @p30='?' (Size = 255), @p31='?' (Size = 4000), @p32='?' (DbType = Int32), @p33='?' (Size = 4000), @p34='?' (Size = 255), @p35='?' (Size = 4000), @p36='?' (DbType = Int32), @p37='?' (Size = 4000), @p38='?' (Size = 255), @p39='?' (Size = 4000), @p40='?' (DbType = Int32), @p41='?' (Size = 4000), @p42='?' (Size = 255), @p43='?' (Size = 4000), @p44='?' (DbType = Int32), @p45='?' (Size = 4000), @p46='?' (Size = 255), @p47='?' (Size = 4000), @p48='?' (DbType = Int32), @p49='?' (Size = 4000), @p50='?' (Size = 255), @p51='?' (Size = 4000), @p52='?' (DbType = Int32), @p53='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p3, @p4, @p5);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p6, @p7, @p8);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p9, @p10, @p11);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p12, @p13, @p14);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p15, @p16, @p17);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p18, @p19);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p20, @p21);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p22, @p23);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p24, @p25);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p26, @p27);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p28, @p29);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p30, @p31, @p32, @p33);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p34, @p35, @p36, @p37);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p38, @p39, @p40, @p41);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p42, @p43, @p44, @p45);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p46, @p47, @p48, @p49);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p50, @p51, @p52, @p53);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:27.9152367+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Nathan@yesenia.net' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Nathan@yesenia.net' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Nathan@yesenia.net' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Nathan@yesenia.net' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:27.9857543+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Chaim_McDermott@dana.io (e813ef88)
2020-12-20T17:02:28.0004067+00:00 0HM5513KNVGOQ:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (Size = 4000), @p9='?' (Size = 4000), @p10='?' (Size = 4000), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 4000), @p14='?' (Size = 4000), @p15='?' (Size = 4000), @p16='?' (Size = 4000), @p17='?' (Size = 4000), @p18='?' (Size = 4000), @p19='?' (Size = 4000), @p20='?' (Size = 4000), @p21='?' (DbType = Decimal), @p22='?' (DbType = Decimal), @p23='?' (DbType = Decimal), @p24='?' (DbType = Decimal), @p25='?' (DbType = Decimal), @p26='?' (DbType = Decimal), @p27='?' (DbType = Decimal), @p28='?' (DbType = Decimal), @p29='?' (DbType = Decimal), @p30='?' (DbType = Decimal), @p31='?' (DbType = Decimal), @p32='?' (DbType = Decimal), @p33='?' (DbType = Decimal), @p34='?' (DbType = Decimal), @p35='?' (Size = 255), @p36='?' (Size = 4000), @p37='?' (DbType = Int32), @p38='?' (Size = 4000), @p39='?' (Size = 255), @p40='?' (Size = 4000), @p41='?' (DbType = Int32), @p42='?' (Size = 4000), @p43='?' (Size = 255), @p44='?' (Size = 4000), @p45='?' (DbType = Int32), @p46='?' (Size = 4000), @p47='?' (Size = 255), @p48='?' (Size = 4000), @p49='?' (DbType = Int32), @p50='?' (Size = 4000), @p51='?' (Size = 255), @p52='?' (Size = 4000), @p53='?' (DbType = Int32), @p54='?' (Size = 4000), @p55='?' (Size = 255), @p56='?' (Size = 4000), @p57='?' (DbType = Int32), @p58='?' (Size = 4000), @p59='?' (Size = 255), @p60='?' (Size = 4000), @p61='?' (DbType = Int32), @p62='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p0, @p1, @p2);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p3, @p4, @p5);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p6, @p7, @p8);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p9, @p10, @p11);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p12, @p13, @p14);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p15, @p16, @p17);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Company` (`Bs`, `CatchPhrase`, `Name`)
VALUES (@p18, @p19, @p20);
SELECT `Id`
FROM `Company`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p21, @p22);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p23, @p24);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p25, @p26);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p27, @p28);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p29, @p30);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p31, @p32);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `Geo` (`Lat`, `Lng`)
VALUES (@p33, @p34);
SELECT `Id`
FROM `Geo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p35, @p36, @p37, @p38);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p39, @p40, @p41, @p42);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p43, @p44, @p45, @p46);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p47, @p48, @p49, @p50);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p51, @p52, @p53, @p54);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p55, @p56, @p57, @p58);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();

INSERT INTO `UserInfo` (`Email`, `Name`, `SourceId`, `Username`)
VALUES (@p59, @p60, @p61, @p62);
SELECT `Id`
FROM `UserInfo`
WHERE ROW_COUNT() = 1 AND `Id` = LAST_INSERT_ID();" (0723d8ff)
2020-12-20T17:02:28.0108231+00:00 0HM5513KNVGOQ:00000004 [ERR] An exception occurred in the database while saving changes for context type '"MutantTest.Infra.Repository.CoreContext"'."
""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Sherwood@rosamond.me' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Sherwood@rosamond.me' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)" (170f029e)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Sherwood@rosamond.me' for key 'UserInfo.IX_UserInfo_Email'
 ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry 'Sherwood@rosamond.me' for key 'UserInfo.IX_UserInfo_Email'
   at MySqlConnector.Core.ResultSet.ReadResultSetHeaderAsync(IOBehavior ioBehavior) in /_/src/MySqlConnector/Core/ResultSet.cs:line 51
   at MySql.Data.MySqlClient.MySqlDataReader.ActivateResultSet() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 130
   at MySql.Data.MySqlClient.MySqlDataReader.NextResultAsync(IOBehavior ioBehavior, CancellationToken cancellationToken) in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 94
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult() in /_/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs:line 27
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at MutantTest.Infra.Repository.UserRepository.InsertUserList(IEnumerable`1 userList) in C:\projects\teste_mutant\MutantTest\MutantTest.Infra\Repository\UserRepository.cs:line 31
2020-12-20T17:02:28.0865679+00:00 0HM5513KNVGOQ:00000004 [WRN] Tentativa de inserir duplicata: Rey.Padberg@karina.biz (46feb013)
2020-12-20T17:02:28.1236233+00:00 0HM5513KNVGOQ:00000004 [INF] Executing ObjectResult, writing value of type '"System.String"'. (8a1b66c8)
2020-12-20T17:02:28.1249978+00:00 0HM5513KNVGOQ:00000004 [INF] Executed action "MutantTest.API.Controllers.UsersController.SaveUsers (MutantTest.API)" in 1657.0135ms (afa2e885)
2020-12-20T17:02:28.1251958+00:00 0HM5513KNVGOQ:00000004 [INF] Executed endpoint '"MutantTest.API.Controllers.UsersController.SaveUsers (MutantTest.API)"' (99874f2b)
2020-12-20T17:02:28.1282552+00:00 0HM5513KNVGOQ:00000004 [INF] Request finished in 1683.2173ms 409 text/plain; charset=utf-8 (791a596a)
2020-12-20T20:34:56.0936727+00:00  [INF] Now listening on: "http://[::]:80" (d826f4b8)
2020-12-20T20:34:56.0985295+00:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-20T20:34:56.1010480+00:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-20T20:34:56.1046825+00:00  [INF] Content root path: "/app" (b5d60022)
2020-12-20T20:35:18.6231775+00:00 0HM554QRO891E:00000003 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:35:18.6269454+00:00 0HM554QRO891E:00000003 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:35:18.7818526+00:00 0HM554QRO891E:00000003 [INF] Received HTTP response after 145.7706ms - OK (6350df9e)
2020-12-20T20:35:18.7858845+00:00 0HM554QRO891E:00000003 [INF] End processing HTTP request after 191.8323ms - OK (7656b38e)
2020-12-20T20:35:18.9952181+00:00 0HM554QRO891E:00000003 [INF] Lista usuarios recuperada com sucesso. (0a6899ef)
2020-12-20T20:36:38.7543974+00:00 0HM554QRO891E:00000004 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:36:38.7572295+00:00 0HM554QRO891E:00000004 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:36:38.7885616+00:00 0HM554QRO891E:00000004 [INF] Received HTTP response after 31.2137ms - OK (6350df9e)
2020-12-20T20:36:38.7924649+00:00 0HM554QRO891E:00000004 [INF] End processing HTTP request after 37.5654ms - OK (7656b38e)
2020-12-20T20:36:39.2596277+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Shanna@melissa.tv (61d3c62c)
2020-12-20T20:36:39.2732224+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Nathan@yesenia.net (9cfcf44a)
2020-12-20T20:36:39.2817140+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Lucio_Hettinger@annie.ca (83ae092c)
2020-12-20T20:36:39.2898776+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Telly.Hoeger@billy.biz (d08b46fa)
2020-12-20T20:36:39.2985466+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Sherwood@rosamond.me (71465a0e)
2020-12-20T20:36:39.3066392+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Chaim_McDermott@dana.io (12a73002)
2020-12-20T20:36:39.3158189+00:00 0HM554QRO891E:00000004 [INF] Novo usuario inserido: Rey.Padberg@karina.biz (ceb43220)
2020-12-20T20:36:54.7003505+00:00 0HM554QRO891E:00000005 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:36:54.7039952+00:00 0HM554QRO891E:00000005 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:36:54.7335144+00:00 0HM554QRO891E:00000005 [INF] Received HTTP response after 28.979ms - OK (6350df9e)
2020-12-20T20:36:54.7379245+00:00 0HM554QRO891E:00000005 [INF] End processing HTTP request after 38.4282ms - OK (7656b38e)
2020-12-20T20:36:54.9153973+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Shanna@melissa.tv (62268090)
2020-12-20T20:36:54.9790954+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Nathan@yesenia.net (45d3b3f6)
2020-12-20T20:36:55.0362124+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Lucio_Hettinger@annie.ca (cd56b31c)
2020-12-20T20:36:55.0821361+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Telly.Hoeger@billy.biz (f62dd99c)
2020-12-20T20:36:55.1393174+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Sherwood@rosamond.me (30f174b5)
2020-12-20T20:36:55.1943813+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Chaim_McDermott@dana.io (e813ef88)
2020-12-20T20:36:55.2612212+00:00 0HM554QRO891E:00000005 [WRN] Tentativa de inserir duplicata: Rey.Padberg@karina.biz (46feb013)
2020-12-20T20:37:12.5638989+00:00 0HM554QRO891E:00000006 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:37:12.5677285+00:00 0HM554QRO891E:00000006 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:37:12.5987990+00:00 0HM554QRO891E:00000006 [INF] Received HTTP response after 31.5138ms - OK (6350df9e)
2020-12-20T20:37:12.6014966+00:00 0HM554QRO891E:00000006 [INF] End processing HTTP request after 39.8413ms - OK (7656b38e)
2020-12-20T20:37:12.6053256+00:00 0HM554QRO891E:00000006 [INF] Lista usuarios recuperada com sucesso. (0a6899ef)
2020-12-20T20:39:49.1511471+00:00 0HM554QRO891E:00000007 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:39:49.1547292+00:00 0HM554QRO891E:00000007 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:39:49.2318822+00:00 0HM554QRO891E:00000007 [INF] Received HTTP response after 76.5293ms - OK (6350df9e)
2020-12-20T20:39:49.2358769+00:00 0HM554QRO891E:00000007 [INF] End processing HTTP request after 84.8049ms - OK (7656b38e)
2020-12-20T20:39:49.3819633+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Shanna@melissa.tv (62268090)
2020-12-20T20:39:49.4447173+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Nathan@yesenia.net (45d3b3f6)
2020-12-20T20:39:49.5055673+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Lucio_Hettinger@annie.ca (cd56b31c)
2020-12-20T20:39:49.5797842+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Telly.Hoeger@billy.biz (f62dd99c)
2020-12-20T20:39:49.6576727+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Sherwood@rosamond.me (30f174b5)
2020-12-20T20:39:49.7352105+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Chaim_McDermott@dana.io (e813ef88)
2020-12-20T20:39:49.8113309+00:00 0HM554QRO891E:00000007 [WRN] Tentativa de inserir duplicata: Rey.Padberg@karina.biz (46feb013)
2020-12-20T20:41:42.9444392+00:00  [INF] Now listening on: "http://[::]:80" (d826f4b8)
2020-12-20T20:41:42.9496833+00:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-20T20:41:42.9532578+00:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-20T20:41:42.9567075+00:00  [INF] Content root path: "/app" (b5d60022)
2020-12-20T20:42:05.9946567+00:00 0HM554UKSFH7S:00000004 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:42:05.9990579+00:00 0HM554UKSFH7S:00000004 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:42:06.1174571+00:00 0HM554UKSFH7S:00000004 [INF] Received HTTP response after 111.2982ms - OK (6350df9e)
2020-12-20T20:42:06.1218241+00:00 0HM554UKSFH7S:00000004 [INF] End processing HTTP request after 143.7137ms - OK (7656b38e)
2020-12-20T20:42:06.2958652+00:00 0HM554UKSFH7S:00000004 [INF] Lista usuarios recuperada com sucesso. (0a6899ef)
2020-12-20T20:42:16.1911412+00:00 0HM554UKSFH7S:00000005 [INF] Start processing HTTP request "GET" https://jsonplaceholder.typicode.com/users (338f1c77)
2020-12-20T20:42:16.1947375+00:00 0HM554UKSFH7S:00000005 [INF] Sending HTTP request "GET" https://jsonplaceholder.typicode.com/users (2e7ac211)
2020-12-20T20:42:16.2228116+00:00 0HM554UKSFH7S:00000005 [INF] Received HTTP response after 28.0545ms - OK (6350df9e)
2020-12-20T20:42:16.2254983+00:00 0HM554UKSFH7S:00000005 [INF] End processing HTTP request after 36.7516ms - OK (7656b38e)
2020-12-20T20:42:16.7614870+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Shanna@melissa.tv (62268090)
2020-12-20T20:42:16.8322148+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Nathan@yesenia.net (45d3b3f6)
2020-12-20T20:42:16.8718188+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Lucio_Hettinger@annie.ca (cd56b31c)
2020-12-20T20:42:16.9092904+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Telly.Hoeger@billy.biz (f62dd99c)
2020-12-20T20:42:16.9504644+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Sherwood@rosamond.me (30f174b5)
2020-12-20T20:42:16.9921343+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Chaim_McDermott@dana.io (e813ef88)
2020-12-20T20:42:17.0354108+00:00 0HM554UKSFH7S:00000005 [WRN] Tentativa de inserir duplicata: Rey.Padberg@karina.biz (46feb013)
